From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Ryan Volz <ryan.volz@gmail.com>
Date: Mon, 8 Dec 2025 12:35:39 -0500
Subject: [PATCH] Add missing header to define std::max

---
 src/traceplot.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/traceplot.cpp b/src/traceplot.cpp
index 7c540f3..a43efa2 100644
--- a/src/traceplot.cpp
+++ b/src/traceplot.cpp
@@ -17,6 +17,7 @@
  *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
+#include <algorithm>
 #include <QPixmapCache>
 #include <QTextStream>
 #include <QtConcurrent>
-- 
2.51.0

